contrib/ceph/micro-osd.sh

```diff
@@ -0,0 +1,90 @@
+#
+#    Copyright (C) 2013,2014 Loic Dachary <loic@dachary.org>
```

Comment:
@noahdesu yes thanks, I'll dig this, I successfully executed the setup on a clean ubuntu install earlier today.